.github/workflows/spark-ci.yml

```diff
@@ -99,6 +99,33 @@ jobs:
         path: |
           **/build/testlogs
 
+  spark-31-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        jvm: [8, 11]
+        spark: ['3.1']
+    env:
+      SPARK_LOCAL_IP: localhost
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.jvm }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - run: echo -e "$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)\t$(hostname -f) $(hostname -s)" | sudo tee -a /etc/hosts
+      - run: ./gradlew -DsparkVersions=${{ matrix.spark }} -DhiveVersions= -DflinkVersions= :iceberg-spark:iceberg-spark-3.1:check :iceberg-spark:iceberg-spark-3.1-extensions:check :iceberg-spark:iceberg-spark-3.1-runtime:check -Pquick=true -x javadoc
+      - uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: test logs
+          path: |
+            **/build/testlogs
```

Comment:
Instead of adding another job here, can we update the `matrix.spark` parameter in the `spark-32-tests` config? One of the reasons to add `3.2` to the module names instead of some derivation like `32` was so that we could build the module names from `matrix.spark`.

Reply:
Ok, I'll try that. I'm not familiar with the CI. What is not clear to me is:
Suppose we have two values in
```
        spark: ['3.1', '3.2']
```
instead of the one value
```
        spark: ['3.1']
```
then in the
```
      - run: ./gradlew -DsparkVersions=${{ matrix.spark }} ...
```
does that get run *twice*, each time with one of the values? Or is that run once, with two values in `sparkValues` (because sparkVersions can take a comma-separated list of versions)?

Reply:
Each run uses just one of the values in the build matrix. So for jvm: [8, 11] and spark: ['3.1', '3.2'] you'll get (8, 3.1), (11, 3.1), (8, 3.2), (11, 3.2).